ios/RNCWKWebView.m

```diff
@@ -718,6 +719,25 @@ - (void)                  webView:(WKWebView *)webView
   decisionHandler(WKNavigationResponsePolicyAllow);
 }
 
+-(void)webView:(WKWebView *)webView decidePolicyForNavigationResponse:(WKNavigationResponse *)navigationResponse decisionHandler:(void (^)(WKNavigationResponsePolicy))decisionHandler
```

Comment:
`decidePolicyForNavigationResponse` will also be invoked for nested frames. Could this handler avoid reporting those or provide the info with the event?

I've been using `isTopFrame = [response.URL isEqual:webView.URL];` in a local patch to detect this, based on the idea that by the time this is invoked `webView.URL` should have already been updated to be the main document URL.